content/applications/hr/time_off.rst

```diff
@@ -17,13 +17,13 @@ time off) are being used, :ref:`accrual plans <time_off/accrual-plans>` can be c
 :ref:`public holidays <time_off/public-holidays>` can be set.
 
 .. note::
-   Be advised, only users with specific access rights can see all aspects of the *Time Off*
-   application.
+   Be advised, only users with specific access rights can see all aspects of the **Time Off** app.
 
-   All users can access the :guilabel:`My Time Off` and :guilabel:`Overview` sections of the *Time
-   Off* application. All other sections require specific access rights.
+   All users can access the *My Time Off*  (:menuselection:`Time Off app --> My Time --> My Time
+   Off`) and *Overview* (:menuselection:`Time Off app --> Overview`) sections of the **Time Off**
+   app. All other sections require specific access rights.
 
-   To better understand how access rights affect the *Time Off* application, refer to the
+   To better understand how access rights affect the **Time Off** app, refer to the
```

Comment:
For reference, the word "application" is fine. You don't need to say or switch to the word "app" wholeheartedly.